assets/src/components/ladder.tsx

```diff
@@ -114,17 +115,14 @@ const Ladder = ({
             ladderVehicles={ladderVehicles}
           />
         )}
-        {orderedTimepoints.map((timepointId: TimepointId, index: number) => {
+        {orderedTimepoints.map((timepoint: Timepoint, index: number) => {
           const y = timepointSpacingY * index
           return (
-            <LadderTimepoint
-              key={timepointId}
-              timepointId={timepointId}
-              y={y}
-            />
+            <LadderTimepoint key={timepoint.id} timepoint={timepoint} y={y} />
           )
         })}
       </svg>
+      <ReactTooltip />
```

Comment:
Does this only have to be included once, or does it need to be near every place where tooltips appear? If we only need it once, it should probably go on `<LadderPage>` (if we only want it when tooltips are on the screen) or `App` (if it should always be there) instead of on every ladder.

Reply:
This is a good thought, but I just tried it and it does need to be in the same component as it's referenced in.